projects/storefrontstyles/scss/cxbase/blocks/forms.scss

```diff
@@ -205,19 +205,16 @@ input[type='radio'] {
 $theme: '' !default;
 @if $theme == 'calydon' {
   :focus {
-    outline-color: var(--cx-color-visual-focus);
+    @include visible-focus();
```

Comment:
instead of using `:focus` globally, let's use it for each element. any changes/new elements will be affected by this

Reply:
that's the idea, since we lost the browser's default focus color and "glow", we need to re-implement it and standardize the look, let's sync-up to revisit the expectations.